packages/codefixes/src/codefixesMessages.json

```diff
@@ -160,11 +160,6 @@
     "description": "Fixes classes so they are initialized correctly.",
     "diagnostics": [2564]
   },
-  "unusedIdentifier": {
```

Comment:
Disabling codefix "Remove unused __" because it bring more confusion then benefits and it conflicts with some  of the other codfixes. It's safer for engineers to work around unused identifiers.

We can enable this codefix partially in future (ex. remove only unused imports)